lib/sdp/media.ex

```diff
@@ -159,4 +168,68 @@ defmodule Membrane.Protocol.SDP.Media do
   end
 
   defp gen_ports(port_no, _), do: [port_no]
+
+  defp header_fields(media) do
+    [
+      serialize_type(media.type),
+      serialize_ports(media.ports),
+      media.protocol,
+      serialize_fmt(media.fmt)
+    ]
+  end
+
+  defp other_fields(media) do
+    [
+      {"i", :title, nil},
+      {"c", :connection_data, Membrane.Protocol.SDP.ConnectionData},
+      {"b", :bandwidth, Membrane.Protocol.SDP.Bandwidth},
+      {"k", :encryption, Membrane.Protocol.SDP.Encryption},
+      {"a", :attributes, Membrane.Protocol.SDP.Attribute}
+    ]
+    |> Enum.map(&add_types(&1, media))
+    |> Enum.reject(&(&1 == ""))
```

Comment:
Is this necessary? If it is, it probably shouldn't.

Reply:
👍  Agrred with @kanes115